fhir-package-source/package-source.rst

```diff
@@ -0,0 +1,83 @@
+.. _packge_source:
+
+FHIR Package Source
+---------------------------
+
+As of SDK version 4.0 you can use a FHIR Package source to resolve FHIR artifacts like ``StructureDefinitions``, ``ValueSets``, and ``CodeSystems`` from one ore multiple FHIR Package.
+This can be from the FHIR core packages, but it can be from any FHIR Package you like. These for FHIR artifacts are needed for example for validating resources. Or to generate snapshots of profiles.
+
+Initiate a FHIR Package Source 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are multiple ways to create a ``FhirPackageSource`` instance, that include the packages you want to use.
+If you have a local copy of the packages you want to use, you can just specify the package file paths to create your ``FhirPackageSource``.
+
+.. code:: csharp
+
+	var package1 = "c:/files/packages/package1.tgz";
+	var package2 = "c:/files/packages/package2.tgz";
+	FhirPackageSource resolver = new(new string[] { package1, package 2});
+
+However, if you don't have the packages you want to resolve your artifacts from yet, you can specify a FHIR package server, and the names and versions (using an ``@``) of the packages to install the packages locally and create a ``FhirPackageSource`` immediately.
+
+.. code:: csharp
+	
+	var package1 = "hl7.fhir.r3.core.xml@3.0.2";
+	var package2 = "hl7.fhir.r3.expansions@3.0.2";
+	FhirPackageSource resolver = new("https://packages.simplifier.net", new string[] { package1, package 2});
+
+.. note:: You may need an active internet connection to use this. This function will first check if the specified packages are already installed locally before searching for the packages online.
```

Comment:
Maybe we should talk about the location where the packages are cached in this case?